### PR TITLE
Docs: Clean up sort-var

### PR DESCRIPTION
**Pull request by matthewdunsdon**
**What issue does this pull request address?**

Documentation readability

**What changes did you make? (Give an overview)**

* Made incorrect and correct examples for sort-var clearer
* Simplified description of options, _including clarifying the default behavior_.

**Is there anything you'd like reviewers to focus on?**

No.

### CODE REVIEW

**Comment by platinumazure** [comment]
LGTM but would like others to review.

File: docs/rules/sort-vars.md [hunk 1]
```diff
@@ -52,15 +52,14 @@ var c, d, a, e;
 
 ## Options
 
-```
-"sort-vars": [<enabled>, { "ignoreCase": <boolean> }]
-```
+This rule has an object option:
 
-### `ignoreCase`
+* `"ignoreCase": false` (default) checks the case-sensitivity of the variables order
```
Comment (nzakas) [inline docs/rules/sort-vars.md]: Can you combine these bullets into one? We usually do one bullet per option name.

Comment (matthewdunsdon) [inline docs/rules/sort-vars.md]: I've made that change.

**Comment by vitorbal** [comment]
Lgtm. Thanks, @matthewdunsdon!

**Pull request merged**